plugins/vlc-video/vlc-video-source.c

```diff
@@ -540,13 +549,13 @@ static void vlcs_update(void *data, obs_data_t *settings)
 				dstr_cat_ch(&dir_path, '/');
 				dstr_cat(&dir_path, ent->d_name);
 				add_file(c, &new_files.da, dir_path.array,
-						network_caching);
+						network_caching, track_index);
 			}
 
 			dstr_free(&dir_path);
 			os_closedir(dir);
 		} else {
-			add_file(c, &new_files.da, path, network_caching);
+			add_file(c, &new_files.da, path, network_caching, track_index);
```

Comment:
Style:  Line exceeds 80 characters.

```suggestion
			add_file(c, &new_files.da, path, network_caching,
					track_index);
```

Reply:
done